[2024-05-24T00:18:33.941+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-24T00:18:33.953+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.transform_Solar_Flare_Data manual__2024-05-24T00:18:30.441951+00:00 [queued]>
[2024-05-24T00:18:33.957+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.transform_Solar_Flare_Data manual__2024-05-24T00:18:30.441951+00:00 [queued]>
[2024-05-24T00:18:33.957+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-24T00:18:33.964+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_Solar_Flare_Data> on 2024-05-24 00:18:30.441951+00:00
[2024-05-24T00:18:33.968+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=130) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-24T00:18:33.968+0000] {standard_task_runner.py:63} INFO - Started process 136 to run task
[2024-05-24T00:18:33.969+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'transform_Solar_Flare_Data', 'manual__2024-05-24T00:18:30.441951+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/***_dag.py', '--cfg-path', '/tmp/tmpznwh95f4']
[2024-05-24T00:18:33.969+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask transform_Solar_Flare_Data
[2024-05-24T00:18:33.987+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline.transform_Solar_Flare_Data manual__2024-05-24T00:18:30.441951+00:00 [running]> on host 7f18484bb9a3
[2024-05-24T00:18:34.025+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='transform_Solar_Flare_Data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-24T00:18:30.441951+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-24T00:18:30.441951+00:00'
[2024-05-24T00:18:34.025+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-24T00:18:34.126+0000] {log.py:232} WARNING - [2024-05-24 00:18:34,126] [INFO] - Data transformation successful.
[2024-05-24T00:18:34.126+0000] {base_logger.py:19} INFO - Data transformation successful.
[2024-05-24T00:18:34.127+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-24T00:18:34.127+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-24T00:18:34.131+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=transform_Solar_Flare_Data, run_id=manual__2024-05-24T00:18:30.441951+00:00, execution_date=20240524T001830, start_date=20240524T001833, end_date=20240524T001834
[2024-05-24T00:18:34.140+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-24T00:18:34.151+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-24T00:18:34.151+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
